content/en/docs/kubernetes/collector/components.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Important Components for Kubernetes
+linkTitle: Components
+spelling:
+  cSpell:ignore filelog crio containerd logtag gotime iostream varlogpods
+  cSpell:ignore varlibdockercontainers
+---
+
+## Introduction
+
+The [OpenTelemetry Collector](/docs/collector) supports many different receivers
+and processors to facilitate monitoring Kubernetes. This section covers the
+components that are most important for collecting Kubernetes data and enhancing
+it.
+
+Important Components:
+
+- [Filelog Receiver](#filelog-receiver) - used to collect Kubernetes logs and
+  application logs written to stdout/stderr.
+
+For application traces, metrics, or logs, we recommend the
+[OTLP receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver),
+but any receiver that fits your data is appropriate.
+
+## Filelog Receiver
+
+The Filelog Receiver tails and parses logs from files. Although it's not a
+Kubernetes-specific receiver, it is still the de facto solution for collecting
+any logs from Kubernetes.
+
+The Filelog Receiver is composed of Operators that are chained together to
+process a log. Each Operator performs a simple responsibility, such as parsing a
+timestamp or JSON. Configuring a Filelog Receiver is not trivial. If you're
+using the [OpenTelemetry Collector Helm chart](/docs/kubernetes/helm/collector/)
```

Comment:
Consider using section-relative paths since the page is within the same section (here and elsewhere in the page):

```suggestion
using the [OpenTelemetry Collector Helm chart](../../helm/collector/)
```